integrations/ansible/ansible-tower-remediation/README.md

```diff
@@ -0,0 +1,123 @@
+
+This integration includes the following resources:
+
+* `ansible-tower` ([pipeline])
+* `ansible-tower` ([handler])
+* `ansible_tower_host` ([secret])
+* `ansible_tower_token` ([secret])
+* `sensu/sensu-ansible-handler:2.1.0` ([asset])
+
+## Dashboards
+
+<!-- List of supported dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+This integration is compatible with the [Ansible Tower Jobs dashboard][ansible-tower-jobs-dashboard]. The Ansible Tower dashboard will display Ansible Tower job status, started/finished times, job template details, and job output.
+
+![](img/job-dashboard.png)
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Get a Secret Application Token
+
+   [Create an application using the Ansible Tower admin console][ansible-tower-application].
+
+   [Create an application OAuth token using the Ansble Tower admin console][ansible-tower-app-token].
+
+1. Configure Secrets Management
+
+   This integration requires the following [Sensu Secrets][secrets]:
+
+   - `ansible_tower_host` (hostname or IP address of the Ansible Tower API)
+   - `ansible_tower_token` (Ansible Tower application OAuth token)
+
+   _NOTE: this integration creates one or more Sensu Secrets using the "env" provider. The corresponding environment variables need to be set on every sensu-backend in the Sensu deployment. To add the environment variables, please modify `/etc/default/sensu-backend` or `/etc/sysconfig/sensu-backend` and restart the sensu-backend service(s)._
+
+1. Add a `io.sensu.ansible.config.actions` ("remediation action") [annotation] to one or more checks.
+
+   **Example**:
+
+   ```yaml
+   metadata:
+     name: check_nginx
+     labels: {}
+     annotations:
+       io.sensu.ansible.config.actions: |
+         [
+           {
+             "template_name": "Start NGINX",
+             "inventory_id": 2,
+             "limit": "",
+             "occurrences": [ 3 ],
+             "severities": [ 1, 2 ]
+           },
+           {
+             "template_id": 8,
+             "inventory_name": "Web Servers",
+             "limit": "",
+             "occurrences": [ 10 ],
+             "severities": [ 1, 2 ]
+           }
+         ]
+   ```
+
+## Plugins
+
+<!-- Links to any Sensu Integration dependencies (i.e. Sensu Plugins) -->
+
+- [sensu/sensu-ansible-handler][sensu-ansible-handler-bonsai] ([GitHub][sensu-ansible-handler-github])
+
+## Metrics & Events
+
+<!-- List of all metrics or events collected by this integration. -->
+
+This integration does not produce any [metrics].
+
+## Alerts
+
+<!-- List of all alerts generated by this integration. -->
+
+This integration does not produce any events that should be processed by an alert or incident management [pipeline].
```

Comment:
Maybe use some text in the Alerts section pointing out this pipeline integration is meant to process events?  And the by default there are no filters in the pipeline

Reply:
I think the annotation requirement is covered in the setup section, and remediation actions are specifically not alerts. I think we should leave this as-is.